admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -127,10 +127,10 @@
 
       <tbody class="bg-white text-3.5 line-[150%] text-black">
         <% @rows.each do |row| %>
-          <tr class="
-            border-b border-gray-100 last:border-0
-            <%= 'bg-gray-15 text-gray-700' if @fade_row_proc&.call(row) %>
-          ">
+          <tr
+            class="border-b border-gray-100 last:border-0 <%= 'bg-gray-15 text-gray-700' if @fade_row_proc&.call(row) %>"
+            data-action="click-><%= stimulus_id %>#toggleRowSelection"
```

Comment:
For non-standard elements like `<tr>` we must specify the event `click->` in `data-action`.